app/controllers/sessions_controller.rb

```diff
@@ -11,14 +11,30 @@ def create
       # Save the user id inside the browser cookie. This is how we keep the user
       # logged in when they navigate around our website.
       session[:user_id] = user.id
-      redirect_to dashboard_path
+
+      if request.xhr?
+        render json: { message: "Login Successful", status: 204 }
```

Comment:
I don't think this message is ever visible to the user.

Reply:
It's not - I was using that to debug stuff, and Sean and I decided to keep it in case it was later decided to show a message of sorts (this way the logic was already figured out).